bindings/rust/src/lib.rs

```diff
@@ -34,4 +167,54 @@ mod tests {
             false
         );
     }
+
+    #[test]
+    fn parse_wasm() {
+        assert!(parse(&[0x00, 0x61, 0x73, 0x6d, 0x01, 0x00, 0x00, 0x00]).is_ok());
+        assert!(parse(&[0x00, 0x61, 0x73, 0x6d, 0x01, 0x00, 0x00, 0x01]).is_err());
+    }
+
+    #[test]
+    fn instantiate_wasm() {
+        let module = parse(&[0x00, 0x61, 0x73, 0x6d, 0x01, 0x00, 0x00, 0x00]);
+        assert!(module.is_ok());
+        let instance = module.unwrap().instantiate();
+        assert!(instance.is_ok());
+    }
+
+    #[test]
+    fn execute_wasm() {
+        let input = hex::decode("0061736d01000000010e036000006000017f60027f7f017f030504000102000a150402000b0400412a0b0700200020016e0b0300000b").unwrap();
```

Comment:
Add comment with wasm code for this

Reply:
It should be the same as `capi_test:execute`. Would be nice to have #593 too.